src/main/resources/com/arcbees/website/client/resources/css/pages/404.gss

```diff
@@ -12,7 +12,269 @@
     padding-top: 200px;
 }
 
+.img404 {
+    margin-bottom: 35px;
+
+    position: relative;
+    z-index: 1;
+}
+
+.text404 {
+    position: relative;
+    z-index: 99;
+}
+
 .honeyLost {
     font-family: "geometriaBold";
     font-size: 4rem;
+    line-height: 1em;
```

Comment:
From the front-end style guide : line-height should be unit-less, thus making it a multiplier based on the font-size